devel/run-e2e.sh

```diff
@@ -20,12 +20,15 @@ set -o pipefail
 
 # This script will run the end-to-end test suite against an already configured
 # kind cluster.
-# If a cluster does not already exist, create one with 'cluster/create.sh'.
+# If a cluster does not already exist, create one with 'cluster/create-kind.sh'.
 
 SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
 export REPO_ROOT="${SCRIPT_ROOT}/.."
 source "${SCRIPT_ROOT}/lib/lib.sh"
 
+SERVICE_IP_PREFIX="${SERVICE_IP_PREFIX:-10.0.0}"
+export DNS_SERVER="${SERVICE_IP_PREFIX}.16"
```

Comment:
Again, move def to lib.sh :)

Reply:
This comment was resolved, but these vars are still defined here 👀

Reply:
weird... moved now!